SX1276/SX1276_LoRaRadio.cpp

```diff
@@ -562,11 +562,11 @@ void SX1276_LoRaRadio::set_tx_config(radio_modems_t modem, int8_t power,
             _rf_settings.fsk.iq_inverted = iq_inverted;
             _rf_settings.fsk.tx_timeout = timeout;
 
-            fdev = (uint16_t) ((float) fdev / (float) FREQ_STEP);
+            fdev = (uint16_t) ((uint64_t) (fdev * 100000000) / (uint64_t) FREQ_STEP);
```

Comment:
Can we have a define for `100000000`? It is used in multiple places so would be nice to have it as a macro, also if named properly it will improve code readability.

Reply:
Good idea